Library/Formula/xcconf.rb

```diff
@@ -0,0 +1,13 @@
+class Xcconf < Formula
+  homepage "https://github.com/AlexDenisov/xcconf"
+  url "https://github.com/AlexDenisov/xcconf/archive/0.1.1.tar.gz"
+  sha1 "d3768af9ab739c49ac107393ac19047a962e703a"
+
+  def install
+    system "make", "build"
+    bin.install "./build/Release/xcconf"
+  end
+
+  test do
```

Comment:
See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!